docs/rules/prefer-stateless-function.md

```diff
@@ -43,3 +44,14 @@ class Foo extends React.Component {
   }
 }
 ```
+
+The following pattern is not considered warnings:
+
+```js
+class Foo extends React.PureComponent {
```

Comment:
I still think this pattern should be considered a warning - being a PureComponent does not mean this is better as a stateful component.